docs/sources/variables/formatting-multi-value-variables.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Formatting multi-value variables"
+type = "docs"
+[menu.docs]
+weight = 500
++++
+
+# Formatting multi-value variables
+
+Interpolating a variable with multiple values selected is tricky as it is not straight forward how to format the multiple values into a string that
+is valid in the given context where the variable is used. Grafana tries to solve this by allowing each data source plugin to
+inform the templating interpolation engine what format to use for multiple values.
+
+> **Note:** The **Custom all value** option on the variable ,ust be blank for Grafana to format all values into a single string.
```

Comment:
This is ust -> must. What this means is that if leave it blank then the Grafana will concatenate (add together) all the values together in the query. Something like `value1,value2,value3`. If a custom all value is used then instead it will be something like `*` or `all`.